pkg/flare/archive.go

```diff
@@ -247,11 +248,12 @@ func getChecksFromProcessAgent(fb flaretypes.FlareBuilder, getAddressPort func()
 			return
 		}
 
-		err := fb.AddFileFromFunc(filename, func() ([]byte, error) { return getHTTPCallContent(checkURL + checkName) })
+		c := apiutil.GetClient(false) // FIX: get certificates right then make this true
+		err := fb.AddFileFromFunc(filename, func() ([]byte, error) { return apiutil.DoGet(c, checkURL+checkName, apiutil.LeaveConnectionOpen) })
 		if err != nil {
 			fb.AddFile( //nolint:errcheck
-				"process_check_output.json",
-				[]byte(fmt.Sprintf("error: process-agent is not running or is unreachable: %s", err.Error())),
```

Comment:
Any specific reason to remove the "process-agent is not running" error message?

Reply:
I thought the errors themselves were clear enough and this existing message is a bit misleading. Since like what happened with bug, it could be neither.